dask/dataframe/methods.py

```diff
@@ -426,6 +425,7 @@ def concat(
     if len(dfs) == 1:
         return dfs[0]
     else:
+        ignore_order = kwargs.get("ignore_order", False)
```

Comment:
Would suggest using `pop` as we are passing this and `**kwargs` later

```suggestion
        ignore_order = kwargs.pop("ignore_order", False)
```